src/platform_impl/macos/window_delegate.rs

```diff
@@ -550,12 +551,12 @@ extern "C" fn window_did_fail_to_enter_fullscreen(this: &Object, _: Sel, _: id)
             shared_state.target_fullscreen = None;
         });
         if state.initial_fullscreen {
-            let _: () = unsafe {
-                msg_send![*state.ns_window,
+            unsafe {
+                let _: () = msg_send![*state.ns_window,
```

Comment:
I've manually updated all the `let ()`, since I hate them. And we have half of the code with `let _: ()` and `let ()`, the first one is more clear, since it clearly states that we bind the type.